### PR TITLE
evaclip code refine

### PR DESCRIPTION
**Pull request by zhiboniu**
evaclip code refine

### CODE REVIEW

File: paddlevlp/datasets/coco_clip.py [hunk 1]
```diff
@@ -22,17 +22,17 @@
...
     META_INFO = collections.namedtuple(
-        "META_INFO",
-        ("images", "annotations", "images_md5", "annotations_md5"))
+        "META_INFO", ("images", "annotations", "images_md5", "annotations_md5"))
     MD5 = ""
     SPLITS = {
         "train": META_INFO(
             os.path.join("coco", "images"),
-            os.path.join("coco", "annotations/coco_karpathy_train.json"),
+            os.path.join("coco", "annotations/coco_karpathy_train_debug.json"),
```
Comment (jerrywgz) [inline paddlevlp/datasets/coco_clip.py]: remove debug code

File: paddlevlp/datasets/laion_clip.py [hunk 2]
```diff
@@ -11,208 +11,65 @@
...
-        self.get_text_emb = 'open_clip_vit_g_14'
+import random
 
-        worker, num_workers = paddle_worker_info()
-        rank = self.config['data_world_rank']
-        world_size = self.config['data_world_size']
-        total_num_workers = num_workers * world_size
-        global_worker_id = rank * num_workers + worker
+# import braceexpand
+import torch
```
Comment (jerrywgz) [inline paddlevlp/datasets/laion_clip.py]: 去除torch 依赖

File: paddlevlp/datasets/coco_clip.py (hunk 1 above)
Comment (zhiboniu) [reply paddlevlp/datasets/coco_clip.py]: done

File: paddlevlp/datasets/laion_clip.py (hunk 2 above)
Comment (zhiboniu) [reply paddlevlp/datasets/laion_clip.py]: done

**Review by jerrywgz — APPROVED** [review]

**Pull request merged**